tests/test_regexp.cppo.ml

```diff
@@ -22,6 +22,16 @@ let mkloc = Loc.mkloc
 let mknoloc = Loc.mknoloc
 let map_loc f {Loc.txt = x; loc} = {Loc.txt = f x; loc}
 
+#if OCAML_VERSION < (4, 8, 0)
+let pp_location_error = Location.report_error
+#else
+let pp_location_error = Location.print_report
+#endif
+
+#if OCAML_VERSION < (4, 3, 0)
+type ('a,'b) result = Ok of 'a | Error of 'b
+#endif
```

Comment:
Again, either `stdlib-shims` or the `result` package. Not sure why we need that at all though.

Reply:
I'll shadow here as well.  And yes, I didn't need to use `result` in particular here, but using a type like it helped.